pom.xml

```diff
@@ -59,36 +59,36 @@
 
 	<properties>
 		<!-- Dependency Versions -->
-		<spring-cloud-commons.version>1.2.3.RELEASE</spring-cloud-commons.version>
-		<spring-cloud-netflix.version>1.3.2.RELEASE</spring-cloud-netflix.version>
-		<spring-cloud-sleuth.version>1.2.2.RELEASE</spring-cloud-sleuth.version>
+		<spring-cloud-commons.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-commons.version>
+		<!--<spring-cloud-netflix.version>1.3.2.RELEASE</spring-cloud-netflix.version>-->
```

Comment:
Seems like a big omission.

Reply:
We need to start somewhere, migrating all the projects will take some more time, and I can see people wanting to start with Discovery and ConfigMaps, those two bring value on their own. 
Can we consider this as the initial work?

Reply:
Each of the other module can be migrated one by one, by someone that is interested in

Reply:
That's fine, but it looks like you deleted ribbon classes.

Reply:
Yes that's part of Ribbon of netflix, we can port those classes back from the 1.5 branch when we move the netflix integration

Reply:
I don't think they need to be deleted since the module won't be compiled.

Reply:
@spencergibb sorry to be a pain but can you point me to the classes that you are talking about? I might be missing something big. 

I've removed the modules as well. I don't want to have modules that are not being compiled in the repository, I would rather have them in the original branch and do a progressive migration. That gives us an opportunity to review the modules/dependencies/code in detail.